### PR TITLE
toggle stylesheet based on sunrise/sunset

### PR DESCRIPTION
**Pull request by MasterOdin**
Uses the [suncalc](https://github.com/mourner/suncalc) library which gets the times for sunset/sunrise based on date, and then using milliseconds since UNIX epoch to determine which stylesheet to use.

### CODE REVIEW

File: public/js/whiteboard.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
-/* global moment, io */
+/* global moment, io, SunCalc */
+
+const longitude = -73.675770;
+const latitude = 42.729270;
```
Comment (ddbruce) [inline public/js/whiteboard.js]: Is there a simple way we can put these in a server-side env var?

File: public/js/whiteboard.js [hunk 2]
```diff
@@ -39,6 +44,20 @@ function updateDate() {
     const todaysDate = moment();
     document.querySelector('#date').innerHTML = todaysDate.format('D MMM YY');
     document.querySelector('#time').innerHTML = todaysDate.format('HH:mm');
+
+    const times = SunCalc.getTimes(new Date(), latitude, longitude);
+    const now = Date.now();
+    if (
```
Comment (ddbruce) [inline public/js/whiteboard.js]: Please comment what this `if` statement is checking in plain English? I think it's "if the time is between (a half hour after sunrise) and (a half hour after sunset)" but I'm not positive

File: public/js/whiteboard.js (hunk 1 above)
Comment (MasterOdin) [reply public/js/whiteboard.js]: Only way with no other changes would be to make a JSON endpoint to serve these variables, but that's a bit janky. Probably better to just incorporate webpack or something into the process and have the variables get filled in at build time.

File: public/js/whiteboard.js (hunk 2 above)
Comment (MasterOdin) [reply public/js/whiteboard.js]: done

File: public/js/whiteboard.js (hunk 1 above)
Comment (ddbruce) [reply public/js/whiteboard.js]: I think that's a good idea for the future. For now, this works.
